content/en/news/releases/1.9.x/announcing-1.9/_index.md

```diff
@@ -0,0 +1,185 @@
+**Added** an expose istiod example to use TLS termination for multicluster or external Istiod  ([Issue #27976](https://github.com/istio/istio/issues/27976))
+
+- **Improved** 'verify-install' detects Istio injector without control plane.
+  ([Issue #29607](https://github.com/istio/istio/issues/29607))
+
+- **Improved** sidecar injection to automatically specify the `kubectl.kubernetes.io/default-logs-container`. This ensures `kubectl logs`
+defaults to reading the application container's logs, rather than requiring explicitly setting the container.  ([Issue #26764](https://github.com/istio/istio/issues/26764))
+
+- **Improved** the sidecar injector to better utilize pod labels to determine if injection is required. This is not enabled
+by default in this release, but can be tested using `--set values.sidecarInjectorWebhook.useLegacySelectors=false`. ([Pull Request #30013](https://github.com/istio/istio/pull/30013))
+
+- **Removed** support for reading Istio configuration over the Mesh Configuration Protocol (MCP). ([Pull Request #28634](https://github.com/istio/istio/pull/28634))
+
+- **Removed** istioctl convert-ingress command
+  ([Issue #29153](https://github.com/istio/istio/issues/29153))
+
+- **Removed** istioctl experimental multicluster command
+  ([Issue #29153](https://github.com/istio/istio/issues/29153))
+
+- **Removed** istioctl experimental post-install webhook command
+  ([Issue #29153](https://github.com/istio/istio/issues/29153))
+
+- **Removed** istioctl register and deregister commands
+  ([Issue #29153](https://github.com/istio/istio/issues/29153))
+
+- **Updated** `istioctl proxy-config log` to allow filtering logs based on label.
+  ([Issue #27490](https://github.com/istio/istio/issues/27490))
+
+- **Updated** prometheus metrics to include `source_cluster` and `destination_cluster` labels by default for all scenarios. Previously, this was only enabled for multi-cluster scenarios. ([Pull Request #30036](https://github.com/istio/istio/pull/30036))
+
+- **Updated** default access log to include `RESPONSE_CODE_DETAILS` and `CONNECTION_TERMINATION_DETAILS` for proxy version >= 1.9. ([Pull Request #27903](https://github.com/istio/istio/pull/27903))
+
+- **Updated** Kiali addon to the latest version v1.29. ([Pull Request #30438](https://github.com/istio/istio/pull/30438))
+
+- **Updated** the default installation of gateways to not configure clusters for `AUTO_PASSTHROUGH`, reducing memory costs.
+  ([Issue #27749](https://github.com/istio/istio/issues/27749))
+
+## Join the Istio community
+
+Our [Community Meeting](https://github.com/istio/community#community-meeting) happens on the fourth Thursday of the month, at 10 AM Pacific. Due to US Thanksgiving, we've moved this month's meeting forward one week to the 19th of November. If you can't make it, why not join the conversation at [Discuss Istio](https://discuss.istio.io/), or join our [Slack workspace](https://slack.istio.io/)?
```

Comment:
Can remove the comment about moving the meeting due to Thanksgiving.

Reply:
Will remove it, copied from 1.8 release notes. :)

Reply:
Done.